### PR TITLE
Notes feature UI

### PR DESCRIPTION
**Pull request by BartDevTWG**
UI for notes feature 

input1
input2
card2
card1

## Checklist

- [ ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation (if appropriate).

### CODE REVIEW

File: src/chat/components/InputToolbar.tsx [hunk 1]
```diff
@@ -44,50 +23,62 @@ interface InputToolbarProps {
 export default function InputToolbar({ onSubmit }: InputToolbarProps) {
   const classes = useStyles();
   const intl = useIntl();
-  const [message, setMessage] = React.useState('');
 
-  const handleChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
-    setMessage(e.target.value);
+  const [value, setValue] = useState(0);
+
+  const handleTabChange = (_event: React.ChangeEvent<{}>, newValue: number) => {
```
Comment (alex-blair) [inline src/chat/components/InputToolbar.tsx:29]: Curious - why did you put an underscore in the argument name `_event`? Is this a pattern we're following?

File: src/chat/components/cards/NotesCard.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  Divider,
+} from '@material-ui/core';
+import CardHeaderComp from './CardHeader';
+import NotesInput from './NotesInput';
+import NotesList from './NotesList';
+import { useSelector } from 'react-redux';
+import { currentUserSelector, loadingCurrentUserSelector } from '../../redux';
+import { CardDetailSkeleton } from '../LoadingScreens';
+
+const NOTES = [
```
Comment (alex-blair) [inline src/chat/components/cards/NotesCard.tsx:18]: This is only placeholder data until we have the backend endpoint, right?

**Review by alex-blair — APPROVED** [review]
Looks good!

File: src/chat/components/cards/NotesCard.tsx (hunk 2 above)
Comment (BartDevTWG) [reply src/chat/components/cards/NotesCard.tsx:18]: That's correct :)

File: src/chat/components/InputToolbar.tsx (hunk 1 above)
Comment (BartDevTWG) [reply src/chat/components/InputToolbar.tsx:29]: `_` is typically used as a placeholder variable. 
 I wanted to indicate an unused parameter 🙂

**Pull request merged**